concordia/signals/handlers.py

```diff
@@ -33,3 +40,25 @@ def update_asset_status(sender, *, instance, **kwargs):
     instance.asset.transcription_status = new_status
     instance.asset.full_clean()
     instance.asset.save()
+
+
+@receiver(post_save, sender=Asset)
+def send_asset_update(sender, *, instance, **kwargs):
+    channel_layer = get_channel_layer()
+    AsyncToSync(channel_layer.group_send)(
+        "asset_updates",
+        {
+            "type": "asset_update",
+            "asset_slug": instance.slug,
+            "asset_status": instance.transcription_status,
```

Comment:
Are there any other values we want to be able to refresh instantly? The only one which came to mind is the difficulty since the others only change in response to admin actions but maybe there's also a thought to including the updated timestamp in both places to support a most/least-recently-updated sort?

Reply:
Looks like we don't actually have an `updated_on` field for `Asset`. Should we add one?

Reply:
I'm leaning no unless @elainekamlley or @jb thinks we should have that particular sort option.